Makefile

```diff
@@ -0,0 +1,18 @@
+.DEFAULT_GOAL := help
+
+.PHONY: start
+start: ## Start the development environment (use Docker)
+	$(SUDO) docker run \
+		--rm \
+		-v $(shell pwd):/var/www/FreshRSS:z \
+		-p 8080:80 \
+		--name freshrss \
+		freshrss/freshrss:dev-alpine
```

Comment:
This hard-coded image name currently does not work on e.g. ARM. If we want to support ARM automatically, we have a few options:
1. add a metafile to Docker Hub (best)
2. use only locally-built Docker images when using Makefile (probably easiest - I think I would go for that one)
3. include a logic here to pull the correct one (not very portable). Example of logic on Debian (but not very portable):
```sh
arch=`dpkg --print-architecture`
if [[ $arch =~ ^arm ]]; then arch="arm";
elif [[ $arch =~ ^amd64 ]]; then arch="amd64";
else echo "Unsupported architecture" >&2; exit 1; fi
echo "Architecture: $arch"
```

Reply:
I don't know about "metafile" on Docker Hub and can't find any information on it… what have you in mind with that?

Also I didn't thought that developers would develop FRSS on ARM architecture… but I'm still at the "ARM is used essentially for mobile phones" phase and I don't follow actuality about ARM :sweat_smile:

Reply:
We could also choose the image tag via an environment variable. Example:

```console
$ TAG=dev-arm make start
```

I would opt for this simple solution for the moment.

Reply:
@marienfressinaud I've seen PRs on GitHub by people who apparently developed *on Android*. I mean, I'd maybe possibly consider using ARM something or other at some indeterminate point in the future but only if I could run a proper OS.